Validation/Geometry/test/runP_Mtd_cfg.py

```diff
@@ -1,12 +1,13 @@
 import FWCore.ParameterSet.Config as cms
+from Configuration.Eras.Era_Phase2_cff import Phase2
 
-process = cms.Process("PROD")
+process = cms.Process("PROD",Phase2)
```

Comment:
@bsunanda is this the most appropriate modifier for this geometry scenario?

Reply:
This qualifier will work for all MTD configurations. The real issue is the choice of SD which is true for all Phase2 scenarios